src/sass/_scrollable_container.scss

```diff
@@ -0,0 +1,7 @@
+.scrollable-container {
+  .content-details {
+    display: block;
+    overflow: hidden auto;
```

Comment:
```suggestion
    overflow: auto;
```
Otherwise on small screens part of a scrollable table (on the right) is impossible to reach. I think the other containers width need further adjustments so the vertical scroller is only visible on very small screens. With this change it is also displayed on large ones.

Reply:
Seems like the horizontal scroller starts to show when the screen width gets to ~400px. So users on mobile devices will most likely see this. This is an existing issue though, maybe we should consider vertically stack the form inputs with the side menu for smaller screen sizes? At the moment I think the horizontal scroller would show for either the side menu or the form

![Screenshot from 2024-01-16 14-03-13](https://github.com/canonical/lxd-ui/assets/151511568/4e124531-07f8-4e69-a997-9a2fe00ab74e)

Reply:
Check the scrollable table (i.e. resource limits tab in your screenshot) There the vertical scroller is much earlier, even on big screens where there is enough space with this change.
But we should have this change, so the config tables are usable on small screens.

Reply:
Edit: Just need some adjustments, to avoid the vertical scroller for large screens in the config tables.

Reply:
Okay I finally see it, reminder to myself not to use the device view port in dev tools for testing these issues

Reply:
I have set the `overflow-y` behaviour to hidden on the outer container i.e. `forms-content` and only rely on the `scrollable-table` overflow behaviour for vertical scroll. For horizontal scroll, we should rely on the outer container `overflow-x` behaviour or else for smaller screen size we won't be able to scroll to the end of the table.